examples/js/loaders/GLTFLoader.js

```diff
@@ -239,7 +238,12 @@ THREE.GLTFLoader = ( function () {
 
 				case 'spot':
 					lightNode = new THREE.SpotLight( color );
-					lightNode.position.set( 0, 0, 1 );
+					// Handle spotlight properties.
+					var spot = light.spot || {};
+					var innerConeAngle = spot.innerConeAngle ? spot.innerConeAngle : 0;
+					var outerConeAngle = spot.outerConeAngle || Math.PI / 4.0;
```

Comment:
Nit: for consistency with other code here, I would write: 

```js
var innerConeAngle = spot.innerConeAngle !== undefined ? spot.innerConeAngle : 0;
var outerConeAngle = spot.outerConeAngle !== undefined ? spot.outerConeAngle : Math.PI / 4.0;
```

Reply:
Will do.

Also opening a new pull request to clean up the PR.